examples/with-fauna/app/api/entries/route.js

```diff
@@ -0,0 +1,22 @@
+import { getAllEntries, createEntry } from '@/lib/fauna'
+import { NextResponse } from 'next/server'
+
+
+export const GET = async (req, res) => {
```

Comment:
You can likely ditch the route handler all together 😄

Reply:
done